db/migrate/20190612152123_create_expired_vacancy_feedbacks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateExpiredVacancyFeedbacks < ActiveRecord::Migration[5.2]
```

Comment:
Roll this back/ reverse this as this migration is no longer necessary for this PR?